test/integration/scenarios/stress/0.transfer_with_data.js.js

```diff
@@ -14,32 +14,31 @@
 
 'use strict';
 
-var Promise = require('bluebird');
-var lisk = require('lisk-elements').default;
-var accountFixtures = require('../../../fixtures/accounts');
-var randomUtil = require('../../../common/utils/random');
-var waitFor = require('../../../common/utils/wait_for');
-var sendTransactionsPromise = require('../../../common/helpers/api')
+const Promise = require('bluebird');
+const lisk = require('lisk-elements').default;
+const accountFixtures = require('../../../fixtures/accounts');
+const randomUtil = require('../../../common/utils/random');
+const waitFor = require('../../../common/utils/wait_for');
+const sendTransactionsPromise = require('../../../common/helpers/api')
 	.sendTransactionsPromise;
-var confirmTransactionsOnAllNodes = require('../common/stress')
+const confirmTransactionsOnAllNodes = require('../../utils/transactions')
 	.confirmTransactionsOnAllNodes;
 
+const broadcasting = process.env.BROADCASTING !== 'false';
 const constants = __testContext.config.constants;
-var broadcastingDisabled = process.env.BROADCASTING_DISABLED === 'true';
 
-module.exports = function(params) {
-	describe('stress test for type 0 transactions with data @slow', function() {
-		this.timeout(1800000);
-		var transactions = [];
-		var maximum = process.env.MAXIMUM_TRANSACTION || 1000;
-		var waitForExtraBlocks = broadcastingDisabled ? 10 : 4; // Wait for extra blocks to ensure all the transactions are included in the block
+module.exports = function(configurations) {
+	describe('@stress : type 0 transactions with data @slow', () => {
+		let transactions = [];
+		const maximum = process.env.MAXIMUM_TRANSACTION || 1000;
```

Comment:
Consistency, in other test we're using `MAXIMUM` (which still need to be renamed to something better).